Source/WebCore/style/AnchorPositionEvaluator.cpp

```diff
@@ -574,7 +574,7 @@ void AnchorPositionEvaluator::updateAnchorPositioningStatesAfterInterleavedLayou
     auto anchorsForAnchorName = collectAnchorsForAnchorName(document);
 
     for (auto elementAndState : document.styleScope().anchorPositionedStates()) {
-        CheckedRef element = elementAndState.key;
+        Ref element = elementAndState.key;
         auto& state = *elementAndState.value;
         if (state.stage == AnchorPositionResolutionStage::Initial) {
             state.anchorElements = findAnchorsForAnchorPositionedElement(element, state.anchorNames, anchorsForAnchorName);
```

Comment:
... and use `Ref { elementAndState.key }` here instead of `element`. This would be more efficient and this looks like potentially hot code (whether covered by current benchmarks or not).

Reply:
How does that help? That's going to generate the near identical code.

Reply:
Oh, I see. I doubt it matters but we can do that.